apis/infix-en_GB/atrium-api-infix-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/pathAssertions.kt

```diff
@@ -296,6 +296,27 @@ infix fun <T : Path> Expect<T>.toBe(@Suppress("UNUSED_PARAMETER") aRegularFile:
 infix fun <T : Path> Expect<T>.toBe(@Suppress("UNUSED_PARAMETER") aDirectory: aDirectory): Expect<T> =
     _logicAppend { isDirectory() }
 
+/**
+ * Expects that the subject of the assertion (a [Path]) is a relative path;
+ * meaning that the [Path] specified in this instance does not start at system root.
```

Comment:
```suggestion
 * meaning that the [Path] specified in this instance does not start at the file system root.
```

like for `api-fluent`